client/src/Components/Navbar/Navlink/index.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
+import { RouterElement } from '../../../router'
 
-export interface NavlinkProps {
-  page: string
-  route: string
-}
+// export interface NavlinkProps {
+//   page: string
+//   route: string
+// }
 
-export const Navlink: React.FC<NavlinkProps> = ({ page, route }) => {
+export const Navlink: React.FC<RouterElement> = ({ name, path }) => {
   return (
-    <a className='text-white font-serif p-2 hover:opacity-75' href={route}>
-      {page}
-    </a>
+    <li className={name.toLowerCase()}>
```

Comment:
I reckon some of these class names could be removed, since I don't think their used for styling?